studio/components/interfaces/BillingV2/Usage/Infrastructure.tsx

```diff
@@ -133,13 +137,16 @@ const Infrastructure = ({
                         </a>
                       </Link>
                     </Alert>
-                  ) : currentBillingCycleSelected && highestIoBudgetConsumption >= 80 ? (
-                    <Alert withIcon variant="warning" title="IO Budget for today is running out">
+                  ) : currentBillingCycleSelected && highestIoBudgetConsumption >= 100 ? (
```

Comment:
Do we no longer want to show ANY hint/warning if the disk IO is >=80%?

Reply:
Will add additional warnings 👍